drafts/draft-lcurley-moq-hang.md

```
@@ -0,0 +1,234 @@
+~~~
+/room123/alice.hang
+/room123/bob.hang
+/room456/zoe.hang
+~~~
+
+A participant issues an ANNOUNCE_PLEASE message to discover any other participants in the same room.
+The server (relay) will then respond with an ANNOUNCE message for any matching broadcasts, including their own.
+
+For example:
+
+~~~
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)